src/PhpSpec/Listener/MethodNotFoundListener.php

```diff
@@ -70,7 +70,8 @@ public function afterExample(ExampleEvent $event)
             return;
         }
 
-        $classname = get_class($exception->getSubject());
+        $subject = $exception->getSubject();
+        $classname = null === $subject ? get_class() : get_class($subject);
```

Comment:
Yes it seems odd that we use the current class - in what case is that even unset?